lib/libpax/libpax_api.h

```diff
@@ -37,8 +33,8 @@ struct libpax_config_t {
     uint8_t wificounter;                    // set to 0 if you do not want to install the WiFi sniffer
     uint8_t wifi_my_country;                // e.g 0 = "EU", etc. select locale for WiFi RF settings
     uint16_t wifi_channel_switch_interval;  // [seconds/100] -> 0,5 sec.
-    int8_t wifi_rssi_threshold;             // Filter for how strong the signal should be to be counted
-
+    int wifi_rssi_threshold;                // Filter for how strong the wifi signal should be to be counted
+    int ble_rssi_threshold;                 // Filter for how strong the bluetooth signal should be to be counted
```

Comment:
When changing `struct libpax_config_t` we should also update  `struct libpax_config_storage_t` and remove something from the reserved bytes so that we keep `LIBPAX_CONFIG_SIZE` the same.

Reply:
-1 +2 +2 = 3 more bytes needed in storage => resized `uint8_t reserved_end[36]` to `uint8_t reserved_end[33]`